Makefile

```
@@ -211,6 +228,16 @@ release-preflight: ci test-release ## Validate release readiness (set RELEASE_TA
 release-dist: test-release ## Build release artifacts (cross-compile)
 	@AL_VERSION="$(AL_VERSION)" DIST_DIR="$(DIST_DIR)" ./scripts/build-release.sh
 
+.PHONY: release-vuln-check
+release-vuln-check: check-govulncheck ## Scan every release executable for reachable vulnerabilities (set DIST_DIR=dist)
```

checkmake (0.3.2)

[warning] Target body for "release-vuln-check" exceeds allowed length of 5 lines (7).

(maxbodylength)